cumulusci/robotframework/pageobjects/BasePageObjects.py

```diff
@@ -136,6 +138,42 @@ def populate_form(self, *args, **kwargs):
         # library.
         self.salesforce.populate_form(*args, **kwargs)
 
+    @capture_screenshot_on_error
+    def select_dropdown_value(self, label, value):
+        """Sets the value of a dropdown form field from one of the values in the dropdown
+
+        ``label`` represents the label on the form field (eg: Lead Source)
+
+        If a modal is present, the modal will be searched for the dropdown. Otherwise
+        the first matching dropdown on the entire web page will be used.
+        """
+        input_element = self._get_form_element_for_label(label)
+        if input_element is None:
+            raise Exception(f"No form element found with the label '{label}'")
+
+        # SeleniumLibrary's scroll_element_into_view doesn't seem to work
+        # for elements in a scrollable div on Firefox. Javascript seems to
+        # work though.
+        self.selenium.driver.execute_script(
+            "arguments[0].scrollIntoView()", input_element
+        )
+        trigger = input_element.find_element_by_class_name("uiPopupTrigger")
```

Comment:
Fwiw, I suspect these classes are associated with one of the deprecated aura components from the `ui` namespace (https://developer.salesforce.com/docs/atlas.en-us.lightning.meta/lightning/ui_input.htm) and will need to be updated in the future as forms are switched over to LWC implementations.